code/_onclick/hud/map_view.dm

```diff
@@ -55,15 +55,23 @@ INITIALIZE_IMMEDIATE(/atom/movable/screen/map_view)
 	return pop_planes
 
 /atom/movable/screen/map_view/proc/hide_from(mob/hide_from)
-	hide_from?.canon_client.clear_map(assigned_map)
-	var/client_ref = WEAKREF(hide_from?.canon_client)
+	var/client/target = hide_from?.canon_client
+	if(!target)
+		return
+
```

Comment:
is this just reflexive? there's no reason to nullcheck here if we do so later

Reply:
The way I previously had this organized it was stopping a potential runtime